src/integrationTest/java/com/mongodb/kafka/connect/MongoSinkConnectorTest.java

```diff
@@ -41,10 +41,11 @@
     @Test
     @DisplayName("Ensure simple producer sends data")
     void testASimpleProducerSmokeTest() {
-        KAFKA.createTopic(getTopicName());
+        String topicName = getTopicName();
```

Comment:
`getTopicName()` has been improved to stop any topic name clashes which could occasionally cause race conditions when creating / deleting.